### PR TITLE
peer review

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ This feature is currently in beta. For more information, see [Beta Releases](/re
 
 ## 1 Introduction
 
-Use the new Consumed REST service document to send REST requests from Mendix Studio Pro. With this feature, you can build, test, and create data structures to store your requests. 
+Use the new Consumed REST Service document to send REST requests from Mendix Studio Pro. With this feature, you can build, test, and create data structures to store your requests.
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:16]: This seems to be a little inconsistent, but other documents capitalize "Service" in similar cases. (e.g. https://docs.mendix.com/refguide/business-event-services/). You can ignore it if you don't think that's correct.

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ You can use this Consumed REST service document to do the following:
     * `PUT`
     * `PATCH`
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:35]: The list of methods isn't really a limitation. Only the note on PATCH methods belongs in this section, the rest should be moved/is already present under use cases.

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 3]
```diff
@@ -13,13 +13,13 @@ This feature is currently in beta. For more information, see [Beta Releases](/re
 
 ## 1 Introduction
 
-Use the new Consumed REST service document to send REST requests from Mendix Studio Pro. With this feature, you can build, test, and create data structures to store your requests. 
+Use the new Consumed REST Service document to send REST requests from Mendix Studio Pro. With this feature, you can build, test, and create data structures to store your requests. 
 
 This feature is supported for [Mendix Studio Pro 10.6](/releasenotes/studio-pro/10.6/) and above.
 
 ### 1.1 Use Cases
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:20]: Would it be possible to give some examples of why someone might use this? I'm not sure how different this is from other APIs, and I don't really know what a "service document" is. Maybe a link or a more detailed explanation of the use cases would help. The list of methods that one can use is more of a capability than a use case.

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 4]
```diff
@@ -13,13 +13,13 @@ This feature is currently in beta. For more information, see [Beta Releases](/re
 
 ## 1 Introduction
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:14]: How is this feature different from this one, also called Consumed Rest Services? https://docs.mendix.com/refguide/consumed-rest-services/ Why is it not in the same node? Perhaps some explanation would clarify.

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 5]
```diff
@@ -46,9 +46,9 @@ You can use this Consumed REST service document to do the following:
 
 ## 2 Installation {#installation}
 
-Download [Studio Pro]( https://marketplace.mendix.com/link/studiopro/) and add the Consumed REST service document to your app. To do this,
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:49]: Is this really installation if you're just adding something from a menu?

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 6]
```diff
@@ -88,62 +88,60 @@ You can configure basic authentication to use for all requests in your document.
...
-2. Name your parameter, add a test value, and click **OK**.
+1. In the **Parameters** tab, click **Add**.
+2. Enter a **Name** and a **Test value** for your parameter, and then click **OK**.
 
     {{< figure src="/attachments/refguide/modeling/integration/consumed-rest-services-beta/parameter.png" >}}
 
-3. Click **Send** to test the parameters. 
+3. To test the parameters, click **Send**. 
 
 {{% alert color="info" %}}
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:110]: What does this mean? Does the user need to do something about it? This section is a sub-section of Configuration, so where exactly are the parameters not supported?

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 7]
```diff
@@ -88,62 +88,60 @@ You can configure basic authentication to use for all requests in your document.
...
 Parameters are not supported in the [configuration](#configuration), including authentication and Base URL.
 {{% /alert %}} 
 
 ### 3.4 Adding Headers {#add-headers}
 
-{{% alert color="info" %}}
-The support for parameter in the header value was introduced in Mendix [10.7](/releasenotes/studio-pro/10.7/).
-{{% /alert %}}
+Starting in version [10.7](/releasenotes/studio-pro/10.7/) of Mendix Studio Pro, you can add a header for any HTTP request which you have specified in your document. To add a header, perform the following steps:
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:116]: This is just a personal preference, I prefer this to the note but feel free to ignore.

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 8]
```diff
@@ -88,62 +88,60 @@ You can configure basic authentication to use for all requests in your document.
...
+3. To use the newly-created JSON string as an entity in your domain model, click **Use JSON Snippet**. You can view the body string in the **Body structure** tab.
 
     {{< figure src="/attachments/refguide/modeling/integration/consumed-rest-services-beta/json-body-structure.png" >}}
 
-4. The entity name is prefilled, but you can change it to a custom name. To create an entity, click **Create Entity** > **OK**. Click **Show** to view the entity in your domain model.
+4. Optionally, you can specify a custom name for your entity. 
+5. To create the entity, click **Create Entity** > **OK**. 
+6. To view the entity in your domain model, click **Show**.
 
 ### 3.6 Creating an Entity from the Response {#create-entity}
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:146]: This section should be formatted as a series of steps, like the sections before and after.

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 9]
```diff
@@ -159,11 +157,12 @@ The entity name is prefilled, but you can change it to a custom name. To create
...
+To select a request in the microflow, perform the following steps:
 
 1. Create a new microflow and drag the [Send REST request](/refguide/send-rest-request/) activity into it.
-2. Double-click the activity and click **Select** to choose the request you want to add, then click **Select** > **OK**.
+2. Double-click the activity and click **Select** to choose the request you want to add. 
+3. Click **Select** > **OK**.
 
 {{< figure src="/attachments/refguide/modeling/integration/consumed-rest-services-beta/select-rest-request.png" >}}
 
-If you have defined parameters in the request, they will be added to the activity. Click **Edit** to change the parameter in the microflow. The parameter values in this activity are used by the runtime instead of the test value defined in the request.
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:169]: Is this another step in the same process? Should it be added to the procedure as step 3?

File: content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md [hunk 10]
```diff
@@ -88,62 +88,60 @@ You can configure basic authentication to use for all requests in your document.
...
 
-`POST`, `PUT` and `PATCH` requests support JSON strings as a request body. Add the JSON body snippet to your request by doing the following:
+`POST`, `PUT` and `PATCH` requests support JSON strings as a request body. Add the JSON body snippet to your request by performing the following steps:
 
-1. Click the **Body** tab and add your JSON string.
+1. In the **Body** tab, enter your JSON string.
 
     {{< figure src="/attachments/refguide/modeling/integration/consumed-rest-services-beta/json-example.png" >}}
 
 2. To validate the input, navigate to the **General** field and click **Send**.
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/refguide/modeling/integration/rest-services/consumed-rest-services-beta.md:137]: From the phrasing, it sounds that **General** contains the **Send** button and is therefore not a field. Can you verify?

**Review by katarzyna-koltun-mx — COMMENTED** [review]
@quinntracy Here's the peer review. The file was clear and easy to follow, and there wasn't much that needed correcting, but I have some questions that you may want to look at. Lmk if there's anything you'd like to discuss.

**Comment by quinntracy** [comment]
Because there are conflicts on this branch, I made another to make edits based on this feedback.

**Pull request closed without merging**